tools/config/TR1X_ConfigTool/Resources/Lang/en.json

```diff
@@ -381,6 +386,10 @@
       "Title": "Enable PS Uzi SFX",
       "Description": "Changes the Uzi sound effects to match the PlayStation version."
     },
+    "underwater_music_mode": {
+      "Title": "Underwater music behavior",
+      "Description": "Changes how the music is played underwater.\n- Full: music plays normally while underwater.\n- Quiet: music plays at half volume while underwater.\n- None: no music plays while underwater."
+    },
```

Comment:
@walkawayy It might be worth adding a note for which option is the OG setting.

`Full: music plays normally while underwater (OG TR1).`